data/gamemoded.8.in

```diff
@@ -118,7 +94,7 @@ Behaviour of the config file can be explained by presenting a commented example:
 .RE
 
 .SH SEE ALSO
-systemd(1)
+gamemoderun(1), system(1)
```

Comment:
Shouldn't be used here `gamemoderun(1), systemd(1)`?

Reply:
thanks for spotting this